mix.exs

```diff
@@ -11,10 +11,14 @@ defmodule Khepri.MixProject do
     ]
   end
 
+  def application do
+    [extra_applications: [:elixir]]
+  end
+
   defp deps() do
     [
       # Dependency pinning must be updated in rebar.config too.
-      {:ra, "2.0.4"}
+      {:ra, "2.0.9"}
```

Comment:
It looks like this fell behind a bit. In Elixir it's very common to use the syntax from [`Version`](https://hexdocs.pm/elixir/Version.html) to enforce dependency versions. Typically these follow semantic versioning rules so you might use

```elixir
{:ra, "~> 2.0"}
```

to enforce "ra at >= 2.0.0 and < 3.0.0" or

```elixir
{:ra, "~> 2.0.0"}
```

for "ra at >= 2.0.0 and < 2.1.0". I think rebar handles this version syntax these days as well.

If you want to be exact,

```elixir
{:ra, "== 2.0.9"}
```

would be the typical way to say it.

Reply:
Indeed, despite the "Dependency pinning must be updated in mix.exs too." I wrote in `rebar.config`, I forgot to update `mix.exs`...

Perhaps our `mix.exs` can be improved to read the information it needs from `rebar.config` and `src/khepri.app.src`. I will try to implement that because this would solve the problem of keeping them in sync.

Thanks, I will use `"== 2.0.9"`. The reason I don't want to loosen the version pinning is that I don't follow the development of Ra closely. There are no release notes and we are bad at release engineering in the RabbitMQ team (I try to improve that in Khepri), meaning I'm not 100% sure Ra follows semantic versioning.